src/collections/components/StoryForm/StoryForm.tsx

```diff
@@ -121,78 +163,96 @@ export const StoryForm: React.FC<StoryFormProps & SharedFormButtonsProps> = (
+          );
+          setParserItemExcerptText(data.getItemByUrl.excerpt || '');
+          if (!ogExcerptText) {
+            formik.setFieldValue('excerpt', data.getItemByUrl.excerpt);
+          }
+
+          // Work out the image URL, if any
+          let imageUrl = '';
+
+          if (
+            data.getItemByUrl.topImageUrl &&
+            data.getItemByUrl.topImageUrl.length > 0
+          ) {
+            // Use the publisher's preferred thumbnail image if it exists
+            imageUrl = data.getItemByUrl.topImageUrl;
             setImageSrc(imageUrl);
           } else {
-            // Use the placeholder to display something on the frontend
-            // while the imageUrl field remains empty as set initially
-            setImageSrc('/placeholders/story.svg');
+            // Try the images array - for YouTube, for example, this returns
+            // the correct thumbnail
+            if (data.getItemByUrl.images && data.getItemByUrl.images[0]) {
+              imageUrl = data.getItemByUrl.images[0].src!;
+              setImageSrc(imageUrl);
+            } else {
+              // Use the placeholder to display something on the frontend
+              // while the imageUrl field remains empty as set initially
+              setImageSrc('/placeholders/story.svg');
+            }
           }
-        }
 
-        // Save the normalised imageUrl value in a hidden input field
-        // to upload to S3 later
-        formik.setFieldValue('imageUrl', imageUrl);
+          // Save the normalised imageUrl value in a hidden input field
+          // to upload to S3 later
+          formik.setFieldValue('imageUrl', imageUrl);
 
-        // And we're done!
-        showNotification('Story parsed successfully', 'success');
-      } else {
-        // This is the error path
-        showNotification(`Story couldn't be parsed`, 'error');
-      }
-      // If this is used to add a story and only the URL is visible,
-      // show the other fields now that they contain something
-      // even if the parser can't process the URL at all.
-      setShowOtherFields(true);
-    },
-    onError: (error: ApolloError) => {
-      // Show any other errors, i.e. cannot reach the API, etc.
-      showNotification(error.message, 'error');
-    },
-  });
+          // And we're done!
+          showNotification('Story parsed successfully', 'success');
+        } else {
+          // This is the error path
+          showNotification(`Story couldn't be parsed`, 'error');
+        }
+        // If this is used to add a story and only the URL is visible,
+        // show the other fields now that they contain something
+        // even if the parser can't process the URL at all.
+        setShowOtherFields(true);
+      },
+      onError: (error: ApolloError) => {
+        // Show any other errors, i.e. cannot reach the API, etc.
+        showNotification(error.message, 'error');
+      },
+    });
```

Comment:
Looks like line 180 - 255 is just the previous existing code but got moved down a bit?